c_glib/test/test-numeric-array.rb

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class TestNumericArray < Test::Unit::TestCase
+  include Helper::Buildable
+
+  def test_mean
+    array = build_double_array([1.1, 2.2, nil])
+    assert_equal(1.6500000000000001, array.mean)
```

Comment:
`assert_equal` isn't suitable for floating point number. Because floating point number may have an error.
Could you use `assert_in_delta` https://www.rubydoc.info/gems/test-unit/Test/Unit/Assertions#assert_in_delta-instance_method instead?

Reply:
Thank you for pointing out. I've fixed it.